server/src/test/java/org/opensearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -139,7 +142,8 @@ protected void onSendRequest(long requestId, String action, TransportRequest req
             (node, reason) -> {
                 assert false : node;
             },
-            () -> new StatusInfo(StatusInfo.Status.HEALTHY, "healthy-info")
+            () -> new StatusInfo(StatusInfo.Status.HEALTHY, "healthy-info"),
+            new ClusterManagerMetrics(NoopMetricsRegistry.INSTANCE)
```

Comment:
Can we check if the metric is not set / equal to 0 ?

Reply:
Updated in the revision.